src/lib/game/world/content-queue.js

```diff
@@ -0,0 +1,83 @@
+class ContentQueue {
+
+  constructor(processor, interval = 1, workFactor = 1, minWork = 1) {
```

Comment:
The `interval` for `setTimeout` defaulting to 1 millisecond, is that intentional?